packages/devextreme-react/src/core/__tests__/template.test.tsx

```diff
@@ -42,10 +44,10 @@ function renderTemplate(
   model = model || {};
   container = container || document.createElement('div');
   const { render } = WidgetClass.mock.calls[0][1].integrationOptions.templates[name];
-
-  return render({
+  const result = render({
     container, model, ...(index && { index }), onRendered,
   });
+  return result;
```

Comment:
It seems that it is redundant changes (stayed after debugging and testing)

Reply:
I deleted this change